source/includes/_reference.md

```diff
@@ -2701,6 +2666,10 @@ DELETE /listen/:channel_id/:resource_type/:resource_id
 client.unwatchCorpus('555daefff80f910100d741d6')
 ```
 
+```javascript
+client.unwatchCorpus('555daefff80f910100d741d6', listener);
```

Comment:
I guess you meant `callback` -- it does not have to be the same function as the one used for `watchCorpus`, right?